<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscaProdutosDialog</class>
 <widget class="QDialog" name="BuscaProdutosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Produtos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="filterLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="termoLabel">
       <property name="text">
        <string>Termo de busca:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="termoLineEdit">
       <property name="placeholderText">
        <string>Nome, MI ou INOM...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tipoProdutoLabel">
       <property name="text">
        <string>Tipo de Produto:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="tipoProdutoComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="tipoEscalaLabel">
       <property name="text">
        <string>Escala:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="tipoEscalaComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="projetoLabel">
       <property name="text">
        <string>Projeto:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="projetoComboBox"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="loteLabel">
       <property name="text">
        <string>Lote:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="loteComboBox"/>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultsTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="paginationLayout">
     <item>
      <widget class="QPushButton" name="firstPageButton">
       <property name="text">
        <string>Primeira</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevPageButton">
       <property name="text">
        <string>Anterior</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pageInfoLabel">
       <property name="text">
        <string>Página 1 de 1 (Total: 0 itens)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPageButton">
       <property name="text">
        <string>Próxima</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lastPageButton">
       <property name="text">
        <string>Última</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QLabel" name="pageSizeLabel">
       <property name="text">
        <string>Itens por página:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pageSizeComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="detailsButton">
       <property name="text">
        <string>Ver Detalhes</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportCSVButton">
       <property name="text">
        <string>Exportar CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
